restler/unit_tests/authentication_test_files/unit_test_server_auth_module.py

```diff
@@ -0,0 +1,4 @@
+def acquire_token(log):
+    log("Returning a valid token")
+    return ("{'user1':{}, 'user2':{}}\n" + "Authorization: valid_module_unit_test_token\n" + "Authorization: shadow_unit_test_token")
```

Comment:
>"{'user1':{}, 'user2':{}}\n" + "Authorization: valid_module_unit_test_token\n" + "Authorization: shadow_unit_test_token") [](http://example.com/codeflow?start=12&length=121)

Please split this for readability
One option:
```
        tok_lines = [
            "{'user1':{}, 'user2':{}}"
            "Authorization: valid_module_unit_test_token"
            "Authorization: shadow_unit_test_token"
        ]
        return '\n'.join(tok_lines)
```

Reply:
Thanks for the suggestion, incorporated this